rails6/en/chapter06-improve-json.adoc

```diff
@@ -22,9 +22,9 @@ Now you should be satisfied with all this work. But we still have a lot of work
 }
 ----
 
-However we want an output that does not contain the fields `user_id`, `created_at` and `updated_at`.
+However, we want an output that does not contain the fields `user_id`, `created_at`, and `updated_at`.
```

Comment:
I do not think `,` is necessary

```suggestion
However, we want an output that does not contain the fields `user_id`, `created_at` and `updated_at`.
```

Reply:
One, two, and three.
We need the comma `,` here. 
It's punctuation, you know.

Reply:
Ok I made somes research and it appears that we can add `,` before the `and` in a list or three or more item but it's optionnal https://www.grammarly.com/blog/comma-before-and/ .

So maybe not introduce it 😃

Reply:
Okay as you like :sweat_smile: